contracts/dca/src/handlers/execute.rs

```diff
@@ -30,15 +40,15 @@ fn create_convert_task_internal(
     let task = CronCatTaskRequest {
         interval,
         boundary: None,
-        // TODO?: should it be argument?
         stop_on_fail: true,
         actions: vec![CronCatAction {
-            // #3 remove and explain what is expected
+            // Cron cat action that will get scheduled
+            // #3.1
             msg: wasm_execute(
-                env.contract.address,
-                &ExecuteMsg::from(DCAExecuteMsg::Convert {
-                    dca_id: dca_id.clone(),
-                }),
+                // We want croncat to call this dca contract and execute specific method
+                "",
+                // Note: ExecuteMsg::from(DCAExecuteMsg),
```

Comment:
Any idea how can we better hint that this is different "ExecuteMsg" @CyberHoward

Reply:
I think we should point to the ExecuteEndpoint in the builder section of our docs. There it shows how the execute message is wrapped. And then also point to the app_messages macro that creates the wrapping for the user.